python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -46,50 +55,53 @@ def __init__(
         super().__init__(inputName=input_name)
 
     def resolve_asset(self, _graphene_info):
+        asset_node = check.not_none(
+            self._external_repository.get_external_asset_node(self._asset_key)
+        )
         return GrapheneAssetNode(
             self._repository_location,
             self._external_repository,
-            self._external_repository.get_external_asset_node(self._asset_key),
+            asset_node,
             self._latest_materialization_loader,
         )
 
 
 class GrapheneAssetNode(graphene.ObjectType):
-    id = graphene.NonNull(graphene.ID)
     assetKey = graphene.NonNull(GrapheneAssetKey)
-    description = graphene.String()
-    opName = graphene.String()
-    op = graphene.Field(GrapheneSolidDefinition)
-    jobs = non_null_list(GraphenePipeline)
-    jobNames = non_null_list(graphene.String)
-    repository = graphene.NonNull(lambda: external.GrapheneRepository)
-    dependencies = non_null_list(GrapheneAssetDependency)
-    dependedBy = non_null_list(GrapheneAssetDependency)
-    dependencyKeys = non_null_list(GrapheneAssetKey)
-    dependedByKeys = non_null_list(GrapheneAssetKey)
     assetMaterializations = graphene.Field(
         non_null_list(GrapheneMaterializationEvent),
         partitions=graphene.List(graphene.String),
         beforeTimestampMillis=graphene.String(),
         limit=graphene.Int(),
     )
-    partitionKeys = non_null_list(graphene.String)
-    partitionDefinition = graphene.String()
+    dependedBy = non_null_list(GrapheneAssetDependency)
```

Comment:
worth adding a comment that these are alphabetical so people know to add future attrs in the right place

Reply:
added